tests/test_parser.py

```diff
@@ -22,144 +22,123 @@
                        Urls)  # noqa
 
 
+class MockResponse:
+    def __init__(self, status_code):
+        self.status_code = Mock(status_code=status_code)
+
+
+@patch('requests.Session')
 class TestsGetPage(unittest.TestCase):
 
-    def test_perform_get_request_uses_provided_url_for_request(self):
-        session = Mock(
-            get=MagicMock(
-                return_value=Mock(status_code=200)
-            )
-        )
+    def test_perform_get_request_uses_provided_url_for_request_with_class(
+            self, mock_response):
+        mock_response.get.return_value = MockResponse(200).status_code
```

Comment:
I think that `status_code = 200` can be set as default value in the mock as it will be used like that in most cases, except ones where you are testing for other statuses. The same can be  done with url and data if needed. In test where you'll need to have `None` or any other value you will be able to explicitly set it there.